spie-proceedings.csl

```diff
@@ -6,16 +6,16 @@
     <link href="http://www.zotero.org/styles/spie-proceedings" rel="self"/>
     <link href="http://spie.org/x14101.xml" rel="documentation"/>
     <author>
-      <name>Walter O'Dell</name>
-      <email>wodel@ufl.edu</email>
-      <uri>http://odell/radonc.med.ufl.edu/</uri>
+      <name>Jiangbin Zhang</name>
+      <email>jz400@cam.ac.uk</email>
+      <uri>http://www.oe.phy.cam.ac.uk</uri>
     </author>
```

Comment:
You're welcome to add your name as contributor or author, but you should leave the original author information intact.